Python/optimizer.c

```diff
@@ -1138,3 +1235,530 @@ _Py_Executors_InvalidateAll(PyInterpreterState *interp)
     }
     interp->executor_list_head = NULL;
 }
+
+/* The following table is generated from this script:
```

Comment:
The script also needs the variable UOP_MAX_TRACE_LENGTH to be set. I would add that to the top of the template, e.g. in the form `UOP_MAX_TRACE_LENGTH = 512`.

I also wish this could go into a separate file included here; and I wish that regenerating it could be automated a little bit more. Then maybe we wouldn't need to special-case optimize.c in the c-analyzer config. (But we can easily put this off. Possibly MAX_TRACE_LENGTH never changes again.)

Reply:
In order to support the JIT, we initialize the array at runtime, so the script is no longer needed.